CONTRIBUTING.md

```diff
@@ -4,11 +4,6 @@ Want to hack on openHAB? Awesome! Here are instructions to get you
 started. They are probably not perfect, please let us know if anything
 feels wrong or incomplete.
 
-## Reporting Issues
```

Comment:
This file can only be found on GitHub, where you also can find the Issue tab easily. No need to have links.
Also: No need to link to openHAB core issues, because we have Issue categories now and they already explain that we have different repos.